### DISCUSSION TITLE
Connection refused: Getsockopt

### DISCUSSION THREAD

**Discussion by GUS0K** [ngrok/ngrok#56] [Ideas]
I have a problem, it is that my friend can not enter my minecraft server, he is told either connection refusion: getsockopt, or Unknown server

**Comment by russorat** [comment]
Hi @GUS0K, happy to try to help here, but i need more info. Does this help you? https://ngrok.com/docs/using-ngrok-with/minecraft/

**Comment by russorat** [comment]
I'm going to close this for now. if you're still having issues, please open a new discussion. Thanks!
